src/mainwindow.cpp

```diff
@@ -379,6 +379,12 @@ void MainWindow::parseTranslation()
         ui->translationLanguagesWidget->setAutoLanguage(m_translator->translationLanguage());
     else
         ui->translationLanguagesWidget->setAutoLanguage(QOnlineTranslator::Auto);
+
+    // If window mode is notification, send a notification including the translation result
+    const AppSettings settings;
+    if (this->isHidden() && settings.windowMode() == AppSettings::Notification) {
+        m_trayIcon->showMessage(tr("Translation Result"), ui->translationEdit->toPlainText(), QSystemTrayIcon::NoIcon, 3000);
+    }
```

Comment:
In [Qt Coding style](https://wiki.qt.io/Qt_Coding_Style) curly braces used only when the body of a conditional statement contains more than one line:
```suggestion
    if (this->isHidden() && settings.windowMode() == AppSettings::Notification)
        m_trayIcon->showMessage(tr("Translation Result"), ui->translationEdit->toPlainText(), QSystemTrayIcon::NoIcon, 3000);
```

Also you specified `QSystemTrayIcon::NoIcon` and `3000`. Is there a reason to display a message without icon and only for a 3 sec?
By default function accepts `QSystemTrayIcon::Information` and `10000`.

Reply:
I don't think the icon has much meaning for the translation results, so I removed the icon. Moreover, 10s is too long for quick view, and the translation results will block other applications during display, so I chose 3S temporarily. However, I think we can also add an option of "Notification display time" to the "Settings".

Reply:
Good point. For now I suggest to leave 3 seconds :)

Reply:
In [Qt Coding style](https://wiki.qt.io/Qt_Coding_Style) curly braces used only when the body of a conditional statement contains more than one line.
```suggestion
    if (this->isHidden() && settings.windowMode() == AppSettings::Notification)
        m_trayIcon->showMessage(tr("Translation Result"), ui->translationEdit->toPlainText(), QSystemTrayIcon::NoIcon, 3000);
```

Reply:
> Good point. For now I suggest to leave 3 seconds :)

I meant that such settings would be nice, but I suggest to implement it into another PR.

Reply:
OK, I see. I will revise them and submit them again.

Reply:
To be more clear: we will merge this PR first after your corrections and "Notification display time" feature can be added into a separate PR.

Reply:
Ok, I got it.